domains/xploite3r.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "xploite3r",
+        "email": "xploite3r@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
that ip seems to be for github pages, it requries 4 IPs
- 185.199.108.153
- 185.199.109.153
- 185.199.110.153
- 185.199.111.153

Reply:
how do i fix this?

Reply:
> how do i fix this?

just add the missing 3 IPs

Reply:
Can I add them through the pannel?